R/multivariate_pipeline.R

```diff
@@ -233,7 +235,7 @@ twas_multivariate_weights_pipeline <- function(
     data_driven_prior_matrices = NULL, data_driven_prior_matrices_cv = NULL, canonical_prior_matrices = FALSE, resid_Y,
     mvsusie_max_iter = 200, mrmash_max_iter = 5000,
     signal_cutoff = 0.05, coverage = 0.95, secondary_coverage = c(0.7, 0.5),
-    min_cv_maf = 0.05, max_cv_variants = -1, cv_seed = 999, cv_threads = 1, verbose = FALSE, imiss_cutoff = NULL) {
+    min_cv_maf = 0.05, max_cv_variants = -1, cv_threads = 1, verbose = FALSE, imiss_cutoff = NULL) {
```

Comment:
here i guess we can also specify cv_threads? although we can make it default to 1.

Reply:
`cv_threads` is right after `cv_seed `, there is `cv_threads=1`